css/templates/e-commerce/page-vars.css

```diff
@@ -23,6 +23,11 @@
   --h6: 1rem;
   --h7: .875rem;
 
+  --space-1: .5rem; /* 8px */
```

Comment:
These are copied over from bass-override-vars. For some reason, when using `px2` the values defined in bass-override-vars aren't used but instead the default for basscss. By including them here, it ensures the right values are used.